tests/kafkatest/version.py

```diff
@@ -185,4 +189,10 @@ def get_version(node=None):
 
 # 2.7.x versions
 V_2_7_0 = KafkaVersion("2.7.0")
-LATEST_2_7 = V_2_7_0
+V_2_7_1 = KafkaVersion("2.7.1")
+V_2_7_2 = KafkaVersion("2.7.2")
+LATEST_2_7 = V_2_7_2
```

Comment:
We are currently on 2.7.0, so don't think we need 2.7.1 and 2.7.2

Reply:
Oops. I thought these were related to IBPs. Will fix that.